src/vuestic-theme/vuestic-sass/vuestic-styles.scss

```diff
@@ -1,12 +1,10 @@
 // Reset
 @import "global/reset.scss";
-// We probably don't need normalize.
+// We probably don't need normalize. <- we probably don't need it indeed, however the file itself contains some crucial for the app things (besides ~normalize import)
```

Comment:
Namely?

It might be possible we still need normalize...

Reply:
Looks like some `html` and `body` styles are located there. I'm not sure that those are not overridden anywhere else, but anyway.